Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -117,6 +122,48 @@ module.exports = class DoubleClickToEdit {
 					}
 					//Discord Is One Of Those
 				}, "Enable Replying"),
+
+				React.createElement(this.FormTitle, {
+					tag: "h3",
+				}, "Key to hold when editing a message"),
+				//Allow the user to control when they edit a message
+				React.createElement(this.RadioItem, {
+					value: editActionKeyState,
+					options: [
+						{ name: "None", value: "none" },
+						{ name: "Shift", value: "shift" },
+						{ name: "Ctrl", value: "ctrl" },
+						{ name: "Any", value: "any" }
+					],
+					onChange: (newState) => {
+						this.editActionKey = newState.value;
+						BdApi.saveData(config.info.id, "editActionKey", newState.value);
+						setEditActionKeyState(newState.value);
+					}
+					//Discord Is One Of Those (i have no clue what that means, but pop off ig)
+				}, "Edit ActionKey"),
```

Comment:
```suggestion
				//Edit
				React.createElement(this.SwitchItem, {
					//The state that is loaded with the default value
					value: editEnableModifier,
					note: "Enable modifier for double clicking to edit",
					//Since onChange passes the current state we can simply invoke it as such
					onChange: (newState) => {
						//Saving the new state
						this.doubleClickToEditModifier = newState;
						Data.save(config.info.id, "doubleClickToEditModifier", newState);
						setEditEnableModifier(newState);
					}
					//Discord Is One Of Those
				}, "Enable Edit Modifier"),
				React.createElement(this.FormTitle, {
					disabled: !editEnableModifier,
					tag: "h3"
				}, "Modifer to hold to edit a message"),
				React.createElement(this.RadioItem, {
					disabled: !editEnableModifier,
					value: editModifier,
					options: [
						{ name: "Shift", value: "shift" },
						{ name: "Ctrl", value: "ctrl" },
						{ name: "Alt", value: "alt" }
					],
					onChange: (newState) => {
						this.editModifier = newState.value;
						Data.save(config.info.id, "editModifier", newState.value);
						setEditModifier(newState.value);
					}
				}),
```
For the editing I've got this in mind. Have it be the first option as a sort of category in your function.